RecommendationEngineClient/RecommendationEngineClient/10 Common/Enum/DiscardedMenuChoice.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RecommendationEngineClient._10_Common.Enum
```

Comment:
what doesn _10_common indicate in this namespace

Reply:
@chetanj257 The Common indicated that this folder is for every layer and all the common things used across the layers are stored here. Such as application constants, Api endpoints, Enums, DTO. I have same folder at the server side as well